### DISCUSSION TITLE
about liberty

### DISCUSSION THREAD

**Discussion by workLes** [froii/testSlider#4] [Q&A] [answered]
What libraries do you like in 2023?

**Comment by froii** [comment, marked as answer]
i realy like - 
1  codemirror 6  with   lezer parser  https://lezer-sandbox-1gurlbleb-a61.vercel.app/  also interesting is lexical from react 
2  Chart JS or D3.js . puppeteer ( for creating pdf ) or react pdf . jspdf. 
3 re-resizable but also interesting to do it by yourself , recat color. react-dropzone
3 graphql , apollo, rxjs new react hooks and new routers
4 prettier. storybook, and check how works passport with different conenction types. ssas.odata.mdx.google.smartbook. saml and basic

also you must try - 
Bun + Vite
svgartista
lottiefiles

there are a lot of opportunities to create low code apps in a really nice way

**Reply by workLes** [reply]
i will try it - thank you ) 
i hope. in the future we will have more interesting packages !
